.github/workflows/CI.yml

```diff
@@ -264,7 +264,7 @@ jobs:
               include("docs/documenter_helpers.jl")
               using Oscar
               DocMeta.setdocmeta!(Oscar, :DocTestSetup, Oscar.doctestsetup(); recursive = true)
-              doctest(Oscar)'
+              doctest(Oscar; doctestfilters=Oscar.doctestfilters())'
```

Comment:
That's another thing we should be able to specify in one place. I'll try to talk to @mortenpi and the rest of the Documenter about ways forward with this regard :-)

Reply:
Maybe this could be integrated into `setdocmeta!` once that meta is used for both docstrings and md-pages